dowhy/causal_estimator.py

```diff
@@ -225,6 +227,34 @@ def estimate_effect_naive(self):
         est = np.mean(df_withtreatment[self._outcome_name]) - np.mean(df_notreatment[self._outcome_name])
         return CausalEstimate(est, None, None, control_value=0, treatment_value=1)
 
+    def effect_tt(self, df: pd.DataFrame, *args, **kwargs):
```

Comment:
dowhy's estimate_effect methods support a target_units parameter. If user provides `target_units="att"`, then that indicates effect on the treated. Is it possible to follow the same convention?

Reply:
now that I see, this function is not really estimating effects. can this be a part of the econml class?